packages/ui-components/src/components/UserMessage/UserMessageFooter.js

```diff
@@ -40,19 +40,25 @@ const FooterAvatar = styled(Avatar)`
   height: 1.25rem;
 `;
 
-export const UserMessageFooter = ({ dateTime, user }) => (
-  <FooterContainer>
-    <FooterInner>
-      <FooterText>
-        Last updated on: {format(dateTime, `${DAY_MONTH_YEAR_DATE_FORMAT} - ${AM_PM_DATE_FORMAT}`)}
-      </FooterText>
-      <FlexSpaceBetween>
-        <FooterUser>{user.name}</FooterUser>
-        <FooterAvatar src={user.avatar} />
-      </FlexSpaceBetween>
-    </FooterInner>
-  </FooterContainer>
-);
+export const UserMessageFooter = ({ dateTime, user }) => {
+  const userInitial = user.name.substring(0, 1);
+  return (
+    <FooterContainer>
+      <FooterInner>
+        <FooterText>
+          Last updated on:{' '}
+          {format(dateTime, `${DAY_MONTH_YEAR_DATE_FORMAT} - ${AM_PM_DATE_FORMAT}`)}
+        </FooterText>
+        <FlexSpaceBetween>
+          <FooterUser>{user.name}</FooterUser>
+          <FooterAvatar initial={userInitial} src={user.avatar}>
```

Comment:
The first instance of an avatar uses `user.profileImage`, while this uses `user.avatar` - is there a reason for that?

Reply:
Good pick up. The user message was developed a while ago before the profileImage existed.